vignettes/articles/sliding.Rmd

```diff
@@ -25,18 +25,12 @@ library(purrr)
 
 # Demonstrations of sliding AR and ARX forecasters
 
-A key function from the epiprocess package is `epi_slide()`, which allows the
-user to apply a function or formula-based computation over variables in an
-`epi_df` over a running window of `n` time steps (see the following `epiprocess`
-vignette to go over the basics of the function: ["Slide a computation over
-signal values"](https://cmu-delphi.github.io/epiprocess/articles/slide.html)).
-The equivalent sliding method for an `epi_archive` object can be called by using
-the wrapper function `epix_slide()` (refer to the following vignette for the
-basics of the function: ["Work with archive objects and data
-revisions"](https://cmu-delphi.github.io/epiprocess/articles/archive.html)). The
-key difference from `epi_slide()` is that it performs version-aware
-computations. That is, the function only uses data that would have been
-available as of time t for that reference time.
+Two key functions in the epiprocess package are `epi_slide()`, which allows the
+user to apply a function over a rolling time window in an `epi_df()` (for the
+basics, see [the slide
+vignette](https://cmu-delphi.github.io/epiprocess/articles/slide.html)), and
+`epix_slide()`, which is the equivalent for an `epi_archive` object and allows
+for snapshot-sliding.
```

Comment:
We should define "snapshot-sliding".

Reply:
I'm planning to do a total overhaul of the vignettes with Nat in PRs after. Most of the changes to vignettes here were just to make them compile.

Reply:
Then maybe just retain the text that was there before?

Reply:
I could revert this, I just saw the "`n` time steps" as being out of date and took a shot at editing.

Reply:
Lte's just add back at the end of your changes the text from "The key difference from `epi_slide()`..." to the end of that paragraph.